include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -150,19 +143,26 @@ class CommonOperatorLoader : public ProtobufLoader {
     if (in.dataType == ONNXIFI_DATATYPE_UINT8) {
```

Comment:
nit: Might make sense to move this to a switch statement. Then you can have a single check for `isOffline` like you did for the unquantized case.